### DISCUSSION TITLE
Review of OCR

### DISCUSSION THREAD

**Discussion by AlexeyMatveev686** [ONLYOFFICE/onlyoffice.github.io#238] [Polls]
<img src="https://raw.githubusercontent.com/ONLYOFFICE/onlyoffice.github.io/master/discussions-icons/ocr.svg">

**Comment by ghost** [comment]
[21May SGK Yazisi.pdf](https://github.com/ONLYOFFICE/onlyoffice.github.io/files/15479146/21May.SGK.Yazisi.pdf)

**Reply by Misokados** [reply]
Hello,
Please describe the issue with the provided file.
